docker-gen/config/docker-gen.cfg

```diff
@@ -3,4 +3,4 @@ template = "/code/docker-gen/templates/Caddyfile.tmpl"
 dest = "/etc/caddy/Caddyfile"
 onlyexposed = false
 watch = true
-wait = "500ms:2s"
+wait = "500ms:2s"
```

Comment:
Let's keep all the newlines, that's what our `.editorconfig` specifies.

Reply:
:heavy_check_mark: